pkg/handlers/health_check.go

```diff
@@ -20,13 +21,22 @@ const (
 )
 
 type healthCheck struct {
-	Status status `json:"status"`
+	Status    status `json:"status"`
+	Datetime  string `json:"datetime"`
+	Version   string `json:"version"`
+	Timestamp string `json:"timestamp"`
 }
 
 // Handle handles a web request and returns a healthcheck JSON payload
 func (h HealthCheckHandler) Handle() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
-		js, err := json.Marshal(healthCheck{Status: statusPass})
+		statusReport := healthCheck{
+			Status:    statusPass,
+			Version:   os.Getenv("APPLICATION_VERSION"),
```

Comment:
We shouldn't be using the `os` package or accessing environment variables in the handlers. You can use the viper config, like here: https://github.com/CMSgov/easi-app/blob/master/pkg/server/server.go#L40, and pass these values into the handler. Since it's a health check and not application/user oriented handler, it's probably ok to just pass the viper config variable itself into the handler.